doc/src/modules/combinatorics/fp_groups.rst

```diff
@@ -0,0 +1,185 @@
+Introduction
```

Comment:
I only wanted to say that backquotes could indicate the third level. Their number should be the same as the length of the header.